src/graphql/schema.graphql

```diff
@@ -162,23 +162,29 @@ type GamePieceRangedAttackAction {
   targetGamePiece: GamePiece!
   resolved: Boolean!
   resolvedAttacks: [ResolvedAttack!]
+  totalDamageDealt: Int
+  totalDamageAverage: Float
```

Comment:
The GraphQL schema changes here give the most concise summary of changes in this PR. Basically we're just calculating and returning via API a bit more data about the resolved attacks. Specifically things like: What were my rolls? How much damage would this set of attacks have done on average, and how did my rolls compare to that? Did I whiff or get lucky?